front/app/containers/Admin/projects/project/analysis/Insights/QuestionButton.tsx

```diff
@@ -39,61 +34,33 @@ const QuestionButton = ({ onClick }: { onClick: () => void }) => {
     );
   }, [analysisId, filters, addQuestionPreCheck]);
 
-  const questionPossible = !preCheck?.data.attributes.impossible_reason;
-  const questionAccuracy = preCheck?.data.attributes.accuracy;
+  const tooManyInputs = preCheck?.data.attributes.impossible_reason;
```

Comment:
If we would add other impossible reasons on the backend, I think we risk that the frontend will show the message that there are too many inputs in that case?

Reply:
Good point! 👍